shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/SerializationProperties.hs

```diff
@@ -239,6 +241,26 @@ instance Arbitrary Mock.BHeader where
 instance Crypto c => Arbitrary (HashHeader c) where
   arbitrary = HashHeader <$> genHash (Proxy @c)
 
+-- TODO this
+instance Arbitrary Quantity where
```

Comment:
I think this may give you a bad time. In particular, you don't want to generate negative values in outputs, and this won't distinguish, I think.

Reply:
yes, doing more work on tests now